pkg/provenance/statement.go

```diff
@@ -0,0 +1,172 @@
+/*
+Copyright 2021 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+	Useful links:
+	SLSA Provenance predicate spec: https://slsa.dev/provenance/v0.1
+	In-toto attestation spec: https://github.com/in-toto/attestation/blob/main/spec/README.md
+*/
+
+package provenance
+
+import (
+	"encoding/json"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/pkg/errors"
+	"sigs.k8s.io/release-utils/hash"
+
+	intoto "github.com/in-toto/in-toto-golang/in_toto"
```

Comment:
Non-blocking: Separate import groups

```suggestion
	intoto "github.com/in-toto/in-toto-golang/in_toto"
	"github.com/pkg/errors"

	"sigs.k8s.io/release-utils/hash"
```